.eslintrc

```diff
@@ -0,0 +1,188 @@
+/*
+This is Aribnb widely used linting configuration. https://github.com/airbnb/javascript
```

Comment:
Good enough to start. Eventually I want to use the [prod-module-boilerplate rules](https://github.com/ericelliott/prod-module-boilerplate/blob/master/.eslintrc) for all our projects. If there's something wrong with those rules, we should fix them.

I've already invested significant time in fine-tuning those rules. AirBnB's rules are trying to be a standard for the whole world. We don't need a standard for the whole world, we need a standard that enforces our own coding standards. =)